src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -1,6 +1,10 @@
 @import '~amo/css/styles';
 
-.HeroRecommendation {
+.HeroRecommendation-banner {
+  padding-bottom: 0;
```

Comment:
This needs to be hidden behind the `enableFeatureHeroRecommendation` config flag, I think? If I compare the screen to production with the config flag off, I see that the spacing is out of sync.

Reply:
Is the spacing problem you are seeing here the one at the top of the page (i.e., right above the text reading "Extensions are like apps for your browser."?

Prod:

![Screenshot 2019-09-30 10 31 39](https://user-images.githubusercontent.com/142755/65888449-8b542a00-e36d-11e9-9dec-f66112b2e09b.png)

With this patch and `enableFeatureHeroRecommendation === false`:

![Screenshot 2019-09-30 10 32 43](https://user-images.githubusercontent.com/142755/65888517-a7f06200-e36d-11e9-9359-0c01742fc975.png)

If so, I can see the problem, but I don't think it's related to this style. In fact, when `enableFeatureHeroRecommendation === false` this style isn't even used, because this is for the `HeroRecommendation`, which isn't rendered when `enableFeatureHeroRecommendation === false`. I think this spacing issue has to do with the refactoring of `AppBanner`, which is currently included in prod, bit hidden on the home page when `enableFeatureHeroRecommendation === false` because it's expected to be included in the Hero. I will see if I can fix that, but I just wanted to be sure I am fixing the specific problem you were reporting here.

Reply:
I fixed the spacing problem shown in the screenshots above. Please let me know if that is the one you were commenting on.

Reply:
Is the spacing problem you are seeing here the one at the top of the page (i.e., right above the text reading "Extensions are like apps for your browser."?

Prod:

![Screenshot 2019-09-30 10 31 39](https://user-images.githubusercontent.com/142755/65888449-8b542a00-e36d-11e9-9dec-f66112b2e09b.png)

With this patch and `enableFeatureHeroRecommendation === false`:

![Screenshot 2019-09-30 10 32 43](https://user-images.githubusercontent.com/142755/65888517-a7f06200-e36d-11e9-9359-0c01742fc975.png)

If so, I can see the problem, but I don't think it's related to this style. In fact, when `enableFeatureHeroRecommendation === false` this style isn't even used, because this is for the `HeroRecommendation`, which isn't rendered when `enableFeatureHeroRecommendation === false`. I think this spacing issue has to do with the refactoring of `AppBanner`, which is currently included in prod, bit hidden on the home page when `enableFeatureHeroRecommendation === false` because it's expected to be included in the Hero. I will see if I can fix that, but I just wanted to be sure I am fixing the specific problem you were reporting here.

Reply:
I fixed the spacing problem shown in the screenshots above. Please let me know if that is the one you were commenting on.

Reply:
Yeah, that's the one. I should have included a screenshot, my bad.